### PR TITLE
Change default ad notifications per hour from 5 to 10

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

**Review by moritzhaller — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/243#issuecomment-1079444389.

**Pull request merged**
